packages/create-turbo/src/cli.ts

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+import chalk from "chalk";
+import { Command } from "commander";
+import notifyUpdate from "./utils/notifyUpdate";
+import { turboGradient } from "./logger";
```

Comment:
seems like a candidaate for turbo-utils package

Reply:
Yea agreed, I have on my list to go through and consolidate the logging from the different packages